### PR TITLE
Submitted Robot Framework badge

### PR DESCRIPTION
**Pull request by rajadavidh**
Added Robot Framework:
https://img.shields.io/badge/Robot%20Framework-000000?style=for-the-badge&logo=robot-framework&logoColor=white

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -391,6 +391,7 @@ Badge | URL
 <img src="https://img.shields.io/badge/Redwood-FDF8F6?style=for-the-badge&logo=redwoodjs&logoColor=BF4722" /> | `https://img.shields.io/badge/Redwood-FDF8F6?style=for-the-badge&logo=redwoodjs&logoColor=BF4722`
 <img src="https://img.shields.io/badge/reveal%20js-F2E142?style=for-the-badge&logo=reveal.js&logoColor=000" /> | `https://img.shields.io/badge/reveal.js-F2E142?style=for-the-badge&logo=reveal.js&logoColor=000`
 <img src="https://img.shields.io/badge/rolup%20js-EC4A3F?style=for-the-badge&logo=rollup.js&logoColor=white" /> | `https://img.shields.io/badge/rolup.js-EC4A3F?style=for-the-badge&logo=rollup.js&logoColor=white`
+<img src="https://img.shields.io/badge/Robot%20Framework-000000?style=for-the-badge&logo=robot-framework&logoColor=white" /> | `https://img.shields.io/badge/Robot%20Framework-000000?style=for-the-badge&logo=robot-framework&logoColor=white`
```
Comment (mrhrifat) [inline README.md:394]: Your arrangement with `Alphabetical` order isn't correct. Please rearrange your badge before `Rolup JS`.

**Review by mrhrifat — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (rajadavidh) [reply README.md:393]: @mrhrifat: Thanks for correcting me. Added `Robot Framework` to line 393 before `Rolup JS`

**Review by mrhrifat — APPROVED** [review]
Thanks for your contribution.

**Comment by rajadavidh** [comment]
Most welcome 🙂

**Pull request merged**